global_helpers/panther_greynoise_helpers.py

```diff
@@ -156,6 +190,17 @@ def vpn_service(self, match_field) -> str:
         return deep_get(self.noise, match_field, "vpn_service")
 
     def context(self, match_field) -> dict:
+        if isinstance(deep_get(self.noise, match_field), list):
```

Comment:
It would be more efficient to make the dict first and then just add the 1 ip field.